presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -200,6 +200,11 @@ Specifier Description
 
 .. warning:: The following specifiers are not currently supported: ``%D %U %u %V %X``
 
+.. warning:: Timestamp is truncated to milliseconds
+
+.. note:: ``Fraction of second`` behaves the same way as ``Microseconds`` from the MySQL reference manual (for the ``date_format`` function)
+    Similarly to MySQL, it accepts between 6 and 9 digits of input.
```

Comment:
The documentation as it currently stands looks much better. Thank you!

I just tried `str_to_date` and `date_format` out in MySQL. It looks like MySQL actually allows 1-9 digits instead of 6, but always print 6 digits. And this is also what `appendFractionOfSecond` in Joda does when you say `(6, 9)`. I was the one that suggested that your documentation should say "6-9 digits". It's my fault. Let's fix the documentation again.